src/QsCompiler/Tests.LanguageServer/ProjectLoaderTests.cs

```diff
@@ -68,7 +68,7 @@ public void SupportedTargetFrameworks()
             Assert.IsTrue(loader.IsSupportedQsFramework("netcoreapp2.1"));
             Assert.IsTrue(loader.IsSupportedQsFramework("netcoreapp2.2"));
             Assert.IsTrue(loader.IsSupportedQsFramework("netcoreapp3.0"));
-            Assert.IsTrue(loader.IsSupportedQsFramework("netcoreapp3.1"));
+            Assert.IsTrue(loader.IsSupportedQsFramework("net6.0"));
```

Comment:
It looks from the above as though the project loader should still support older versions, even if they can't be built?

Reply:
Thanks for the catch, Chris! You're entirely right.